lib/metanorma/m3aawg/processor.rb

```diff
@@ -25,9 +25,9 @@ def fonts_manifest
         {
           "Arial" => nil,
           "Cambria Math" => nil,
-          "Courier" => nil,
           "Courier New" => nil,
-          "EB Garamond" => nil,
+          "EB Garamond 08" => nil,
+          "EB Garamond 12" => nil,
```

Comment:
@Intelligent2013 Do we need "EB Garamond 08"? Also, should we change "Garamond" to "EB Garamond 08" in other places?

Reply:
Hmm. It's not simple question... What is difference between EB Garamond 08 and 12?

Reply:
They differ by size by which they look best

> EBGaramond12-Regular: Regular font for design size 12pt
> EBGaramond08-Regular: Regular font for design size 8pt
https://github.com/georgd/EB-Garamond

So if we use 12pt most of the time, it's sufficient to use "EB Garamond 12"?

Reply:
Yes, main font-size in 12pt, so let's use EB Garamond 12.

> Also, should we change "Garamond" to "EB Garamond 08" in other places?

Yes, "Garamond" should be replaced by `EB Garamond 12` here https://github.com/metanorma/mn-native-pdf/blob/master/xslt_src/m3aawg.report.core.xsl

No need to update in mn2pdf/src/main/resources/pdf_fonts_config.xml. Entry for `EB Garamond 12` should be created automatically based on fontist manifest file.

Reply:
Thanks!

Did the changes here https://github.com/metanorma/mn-native-pdf/pull/325/files

Reply:
"EB Garamond 12" should be added also in mn-native-pdf/fonts/manifest.yml

Reply:
Thanks! Added it too.